### PR TITLE
Add Project.toml

### PR DESCRIPTION
**Pull request by EricForgy**
This package has no dependencies yet the `General` registry says it has a dependency on `Pkg`.

https://github.com/JuliaRegistries/General/blob/master/R/Reexport/Deps.toml

I'm guessing the reason is that `Reexport` doesn't have a `Project.toml` and for some reason `Pkg` is being inserted as a dependency.

Hopefully this fixes things because I don't want `Pkg` dependencies in my `Manifest.toml` when nothing in my package depends on `Pkg`.

### CODE REVIEW

**Comment by EricForgy** [comment]
Great minds think alike? 😊 #15

**Comment by EricForgy** [comment]
Note: I removed an old `Compat` from Dec 2017.

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name = "Reexport"
+uuid = "189a3867-3050-52da-a836-e630ba90ab69"
+authors = ["Simon Kornblith <simon@simonster.com>"]
+version = "0.2.0"
```
Comment (ararslan) [inline Project.toml]: Please remove the version; it serves no real purpose and is annoying to keep in sync with the tagged version.

File: .travis.yml [hunk 2]
```diff
@@ -1,14 +1,14 @@
+# Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
 os:
   - linux
   - osx
 julia:
-  - 0.7
```
Comment (ararslan) [inline .travis.yml]: The package still accepts 0.7, so we should continue to test on it. Adding 1.0 and 1.1 is good though.

**Comment by EricForgy** [comment]
Thanks for the feedback. Done 👍😊

File: .travis.yml [hunk 3]
```diff
@@ -1,14 +1,15 @@
+# Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
 os:
   - linux
   - osx
 julia:
   - 0.7
+  - 1.0
```
Comment (ararslan) [inline .travis.yml]:
```suggestion
  - 1.0
  - 1.1
```

**Pull request merged**